app/client/components/FormRenderer.ts

```diff
@@ -258,14 +259,21 @@ abstract class BaseFieldRenderer extends Disposable {
     return dom('label',
       css.label.cls(''),
       css.label.cls('-required', Boolean(this.field.options.formRequired)),
-      {for: this.name()},
+      {for: this.name(), id: `${this.name()}-label`},
```

Comment:
Nitpicking: I don't like this :(. Name suggest that spaces are allowed, but the id attribute doesn't allow spaces in it.
WDYT about adding `public id()` method with default implementation that will strip whitespace from the name?